app/src/main/java/com/duckduckgo/app/browser/pageloadpixel/PageLoadedHandler.kt

```diff
@@ -52,6 +53,7 @@ class RealPageLoadedHandler @Inject constructor(
     @AppCoroutineScope private val appCoroutineScope: CoroutineScope,
     private val dispatcherProvider: DispatcherProvider,
     private val optimizeTrackerEvaluationRCWrapper: OptimizeTrackerEvaluationRCWrapper,
+    private val autoconsentSettingsRepository: AutoconsentSettingsRepository,
```

Comment:
We must only use public APIs to communicate with other modules. Instead you can inject Autoconsent and use isSettingEnabled() to know if the user had the setting enabled.